File: abc.com/src/main/java/application/employee/Employee.java
package application.employee;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import application.common.BasicInfo;

@Document
public class Employee extends BasicInfo{
	
	@Id
	private String id;
	private String name;
	private String email;
	private String skypeId;
	private String companyId;
	
	
	public Employee(String name, String email, String skypeId, String companyId) {
		this.name = name;
		this.email = email;
		this.skypeId = skypeId;
		this.companyId = companyId;
	}
	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public String getSkypeId() {
		return skypeId;
	}
	public void setSkypeId(String skypeId) {
		this.skypeId = skypeId;
	}
	public String getCompanyId() {
		return companyId;
	}
	public void setCompanyId(String companyId) {
		this.companyId = companyId;
	}
	

}

File: abc.com/digi-client1/src/app/employee/employee.ts
import { Class } from '@angular/core';

export class Employee {

  private  id: string;
  private  name: string;
  private  email: string;
  private  skypeId: string;
  private  companyName: string;
  private  companyId: string;

  constructor() {
  }
  
   public setId ( id: string ) {
    this.id = id;
  }
  public getId (): string {
    return this.id ;
  }
  
  public setName ( name: string ) {
    this.name = name;
  }
  
  public getName (): string {
    return this.name ;
  }
  
  public setEmail ( email: string ) {
    this.email = email;
  }
  public getemail (): string {
    return this.email ;
  }
  public setSkypeId ( skypeId: string ) {
    this.skypeId = skypeId;
  }
  public getSkypeId (): string {
    return this.skypeId ;
  }
  public setCompanyName( companyName: string ) {
    this.companyName = companyName;
  }
  public getCompanyName (): string {
    return this.companyName ;
  }
  public setCompanyId( departmentId: string ) {
    this.companyId = departmentId;
  }
  public getCompanyId(): string {
    return this.companyId ;
  }
}
